SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -43,6 +43,8 @@
 #include "LevelEditor.h"
 #include "Misc/FileHelper.h"
 
+#include "Utils/LaunchConfigEditor.h"
```

Comment:
How come this is in it's own include section?

Reply:
What do you mean by "include section" ?